### PR TITLE
Add initial pan positions props

### PR DESCRIPTION
**Pull request by bereziuk**
Hi @ajainarayanan 
Thank you for this nice wrapper. It helps me a lot.

I've added optional props to control initial pan position.
It is passed to your react-pan-zoom component and works fine.
If you need some additional work to approve it, just let me know.

When could I expect merge and new version release?

### CODE REVIEW

**Comment by ajainarayanan** [comment]
@bereziuk Thank you for the pull request! I will try my best to review this tonight (PST). Looks like a small change.

~nit: You don't need to add the built react-dag.js file I think. I remember adding it to the npm publish script.~ Apparently I didn't 🤷‍♂️

File: src/dag.tsx [hunk 1]
```diff
@@ -335,6 +338,13 @@ export default class DAG extends React.Component<IDAGProps, IDAGState> {
           height="100%"
           width="100%"
           zoom={this.state.zoom}
+          pandx={this.props.panPositionX || 0}
+          pandy={this.props.panPositionY || 0}
+          onPan={(x, y) => {
+            if (this.props.onPanMove && typeof this.props.onPanMove === "function") {
+              this.props.onPanMove(x, y)
```
Comment (ajainarayanan) [inline src/dag.tsx]: nit: Missing semicolon.

**Review by ajainarayanan — APPROVED** [review]
Just one minor comment. Otherwise 👍 Please fix and squash the commits.

File: src/dag.tsx (hunk 1 above)
Comment (bereziuk) [reply src/dag.tsx]: Fixed

**Comment by bereziuk** [comment]
@ajainarayanan I have added missing semicolon and PR is ready to merge
Thank you for quick reply 👍

**Comment by ajainarayanan** [comment]
@bereziuk Have published the latest changes https://www.npmjs.com/package/react-dag/v/2.0.0-alpha-3. Thanks for the pull request!

**Pull request merged**